notebooks/00_core.ipynb

```diff
@@ -31,7 +31,11 @@
     "import numpy as np\n",
```

Comment:
<p>care to add a docstring?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>